cumulusci/tasks/bulkdata/utils.py

```diff
@@ -9,11 +10,16 @@
 from sqlalchemy.orm import mapper
 
 from cumulusci.core.exceptions import BulkDataException
-from cumulusci.utils import convert_to_snake_case
 
 
-def get_lookup_key_field(lookup, sf_field):
-    return lookup.get("key_field") or convert_to_snake_case(sf_field)
+def get_lookup_key_field(lookup, model=None):
+    # This should be removed in late 2020.
+    # It just provides some temporary backwards-compatibility
+    # for Snowfakery
```

Comment:
What do you mean about backwards-compatibility for Snowfakery? This function is not used in Snowfakery.

Reply:
I forgot that it is only called indirectly. You think it's a safe bet that nobody is using it externally?

Reply:
I don't see any mention of it in our products and think it's unlikely to be used elsewhere.

Reply:
I removed it: https://github.com/SFDO-Tooling/CumulusCI/pull/1772/commits/766f4612dcb6f99c27ce1f7a9523a4fbee5ac162